library/libraries.js

```diff
@@ -1862,5 +1862,16 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
             
             return false;
         }
+    },
+    'Sugar': {
+        id: 'sugar',
+        icon: 'sugar',
+        url: 'https://sugarjs.com',
+        npm: 'sugar',
+        test: function (win) {
+            if (!win.Sugar) return false;
```

Comment:
Versions up to 1.3.9 don't seem to include a `Sugar` global, but instead patch directly onto prototypes. Eg, `Array.SugarMethods` would exist.

Reply:
Thanks!